public/schema.json

```diff
@@ -69,12 +69,6 @@
       "Asset": {
         "$ref": "refs/components/schemas/Asset.json"
       },
-      "Assets": {
```

Comment:
Need to add back in the schema.json any components that exist in the folder:
`public/refs/components/schemas`